bwmenu

```diff
@@ -49,7 +49,7 @@ DIR="$(dirname "$(readlink -f "$0")")"
 source "$DIR/lib-bwmenu"
 
 ask_password() {
-  mpw=$(rofi -dmenu -p "Master Password" -password -lines 0) || exit $?
+  mpw=$(printf ''|rofi -dmenu -p "Master Password" -password -lines 0) || exit $?
```

Comment:
I'd place spaces around the pipe character here.

```
mpw=$(printf '' | rofi -dmenu …)
```

Reply:
Updated :)